src/interface_adapter/signup/cancel/CancelPresenter.java

```diff
@@ -0,0 +1,33 @@
+package interface_adapter.signup.cancel;
+
+import interface_adapter.ViewManagerModel;
+import interface_adapter.login.LoginState;
+import interface_adapter.login.LoginViewModel;
+import interface_adapter.signup.SignupViewModel;
+import interface_adapter.start.StartViewModel;
+import use_case.signup.cancel.CancelOutputBoundary;
+
+public class CancelPresenter implements CancelOutputBoundary {
+
+    private final ViewManagerModel viewManagerModel;
+    private final CancelViewModel cancelViewModel;
+    private final StartViewModel startViewModel;
+
+    public CancelPresenter (CancelViewModel cancelViewModel,
+                            StartViewModel startViewModel, ViewManagerModel viewManagerModel) {
+        this.viewManagerModel = viewManagerModel;
+        this.cancelViewModel = cancelViewModel;
+        this.startViewModel = startViewModel;
+    }
+    @Override
+    public void prepareSuccessView() {
+//        SignState loginState = loginViewModel.getLoginState();
```

Comment:
Isn't loginState meant to be an instance of LoginState? If not then this line is fine.